.github/ISSUE_TEMPLATE/doc-issue.yml

```diff
@@ -0,0 +1,31 @@
+name: Documentation issue
+description: Highlight a documentation issue
+labels:
+  - kind/doc
+  - needs-triage
+body:
+  - type: textarea
+    id: Inquiry
+    attributes:
+      label: Inquiry
+      description: "Describe what you were looking for in the documentation: "
+      placeholder: ex. "How to bootstrap onto AWS."
+    validations:
+      required: true
+
+  - type: textarea
+    id: Improvement
+    attributes:
+      label: Improvement
+      description: "Describe what improvement you would like to see: "
+      placeholder: ex. "That there is a document the describes bootstraping AWS and links to recommended configuration for production use."
+    validations:
+      required: false
```

Comment:
We need something to be mandatory besides just "what were you looking for". 
The user needs to submit what problem they had.

Reply:
Documentation issues should be in another repo. Maybe we can have this as a "CLI help" bug reporting so it's clear they limit it to things they see in the client? and maybe just redirect them by wording to open a report in https://github.com/juju/docs/issues if they are referring to proper documentation?